app/views/homepage/_search_bar.haml

```diff
@@ -1,3 +1,10 @@
+- # report search usage: keyword search results fetched
+- content_for :extra_javascript do
+  :javascript
+    if(document.getElementById('q').value !== "" && document.getElementById('lc') == null) {
+      window.report_analytics_event('user', "main search", "keyword search");
+    }
+
```

Comment:
So we have two code branches: A) Using keyword search and B) Using location search.

To me it feels weird that these two logically connected code branches are far away from each other.

Would it make sense to add an `else` branch here, which would send the event if location search is in use?

```js
if ( keyword search into use ) {
  send event about keyword search
} else {
  send event about location search
}